src/databricks/labs/ucx/runtime.py

```diff
@@ -23,6 +23,127 @@
 logger = logging.getLogger(__name__)
 
 
+def _get_view_definition(cfg: WorkspaceConfig) -> str:
+    return f"""vw_failure_summary AS WITH failuretab (failures, component) AS (
+        SELECT
+          failures,
+          "jobs" AS component
+        FROM
+          hive_metastore.{cfg.inventory_database}.jobs
```

Comment:
Use $inventory here as well and replace it once with cfg.replace_inventory_variable() (don't remember exact name)